src/App.js

```diff
@@ -0,0 +1,206 @@
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+
+  const handleClick = () => {
+    if (value !== "") {
+      const newItems = [
+        ...items, 
+        {
+          id: v4(),
+          value,
+          isCompleted: false,
+          isHigh: false,
+          isMedium: false,
+        },
+      ];
+  
+      const selectValue = document.querySelector("select")?.value;
+      if (selectValue === "High") {
+        newItems[newItems.length - 1].isHigh = true;
+      } else if (selectValue === "Medium") {
+        newItems[newItems.length - 1].isMedium = true;
+      }
+  
+      setItems(newItems);
+      setValue("");
+      setCounter(counter => counter + 1);
+    }
+  };
+  const handleOnKeyDown = (event) => {
+    if (event.keyCode === 13) {
+      handleClick();
+    }
+  };
+
+  const saveToLocalStorage = () => {
+    localStorage.setItem("items", JSON.stringify(items));
+    localStorage.setItem("counter", JSON.stringify(counter));
+    localStorage.setItem("basicRating", JSON.stringify(basicRating));
+
+    };
+
+ 
+  const markAsCompleted = (id) => {
+    const index = items.findIndex((item) => item.id === id);
+ 
+    const newItem = {
+      ...items[index],
+      isCompleted: !items[index].isCompleted,
+    };
+
+    const newItems = [...items];
+    newItems.splice(index, 1, newItem);
+    setItems(newItems);
+    
+  }
+
+  const dateOfDone = () => {
+    const data = new Date()
+    const newData = data.toLocaleString()
+   const resultData = "Wykonano:" + " " + newData
+    return resultData
+  }
+
+
+
+  const removeItem = (id) => {
+    const index = items.findIndex((item) => item.id == id);
+
+    const newItems = [...items];
+    
+     const newestItems = newItems.filter((e) => e.id !== id)
+
+     setCounter(counter => counter - 1);
+    setItems(newestItems);
+   
+  };
+
+  const onOptionChangeHandler = (event) => {
+    if(event.target.value == 'High') {
+      console.log("Value - ", event.target.value)
+    }
+  }
+
+  const mediumPriorityItems = items.filter((item) => item.isMedium);
+
+
+
+ 
+  return (
+   
+  <div>
+      <InputAndSelect 
+      value={value}
+      handleChange = {handleChange}
+      onOptionChangeHandler={onOptionChangeHandler}
+      handleOnKeyDown={handleOnKeyDown}
+      handleClick={handleClick}/>
+
+      <div className="counter"><Counter count={counter} /></div>
+      <div className="bothLists">
+    
+      <span  className="high-priority-list">
+          <HighPriorityList items={items} />
+      </span>
+      <span>
+      <h2 className="medium-priority-h2">Medium Priority List:</h2>
+        {mediumPriorityItems.map((item) => (
+          <li key={item.id} className="medium-priority">
+        {!item.isCompleted ? item.value : null}
+          </li>
+        ))}
+    </span>
+      <Item 
+      items={items}
+      markAsCompleted={markAsCompleted}
+      removeItem={removeItem}
+      dateOfDone={dateOfDone}
+      CheckIcon={CheckIcon}
```

Comment:
Nie ma potrzeby przekazywania tego jako props. Możesz to zaimportować wewnątrz komponentu.